### PR TITLE
Several fixes lo the landing page

### PR DESCRIPTION
**Pull request by znegrin**
This includes several changes on the home page as requested on the document [Landing Quick fixes & improvements](https://docs.google.com/document/d/1rCasMWwPL-VosmWCd98cXdZsQnBVbVjsuYdQvCbjO9c/edit#). Notice that this time I include several minor changes on the same request as they were all made before knowing the proper process. As requested by @ricardobaeta I added here the first changes all in the same request. It includes:

- Logo update to new one to make it consistent w/ brand style guide. It works perfectly on all pages, on different screen sizes and on scrolling.

- Change CTA button colors

Before: 

image

image

After:

image

image

- Open source block updated with new logo and different distribution of the texts.

Before:

image

After:

image

- Unified Try source{d} Block with Newsletter Sign-Up

Before:

image

After:

image

- Aligned with new logo and & text color guidelines

- Footer color changes, and adaptation of the layout to fit with the new Newsletter block.

Before:

image

After: 

image

Signed-off-by: znegrin <zurinegrin@gmail.com>

### CODE REVIEW

File: src/sass/partials/_layout.scss [hunk 1]
```diff
@@ -451,34 +464,53 @@ section.updates {
 
 // Common - Full CTA
 
+// Common - Full CTA
```
Comment (dpordomingo) [inline src/sass/partials/_layout.scss]: repeated at `#L465` :point_up:

**Comment by dpordomingo** [comment]
@ricready at staging https://landing-staging.srcd.run

**Comment by ricardobaeta** [comment]
@marnovo @campoy This is already up to validation at https://landing-staging.srcd.run/

**Comment by ricardobaeta** [comment]
@znegrin For this to be merged, the only thing I would address is to give some love to the :hover effects on all CTA's. You can play with css brightness. As soon as you update the PR with a good looking solution for this final detail, let's merge!!

**Comment by vcoisne** [comment]
@ricardobaeta @znegrin Does not look like the URLs of the "Install Engine" CTAs are now pointing to https://go.sourced.tech/engine-download ?

**Comment by ricardobaeta** [comment]
@vcoisne thanks for this smart detail :)

**Comment by dpordomingo** [comment]
@marnovo yes: the staging https://landing-staging.srcd.run you visited already contained the latest changes made by #348, #351, #352 and #353.

@ricardobaeta @znegrin feel free to ping me if you update anyone of those PRs and you need to deploy them into `staging` again.

**Comment by znegrin** [comment]
Hi @marnovo @ricardobaeta @dpordomingo 

I just updated with a commit with the remaining things.

Notice that this one: "Google calendar on Community page -> conferences & meetups still has old soft blue color" I couldn't change it as it is an iframe. I checked how Google creates the code and they don't allow many customization further than the size and the background. Please, if you know a way to do that just let me know.

The other changes are: 

- Change the CTA hover effect (I made all the same subtle light hover effect)

image

(Sorry, I think you'll appreciate more on the actual site).

- Social media icons on the community page

image

- Button w/ talent@sourced. on Company page

image

Please let me know if I am missing anything else. I did all on this same PR as it was requested like that. Hope is not a problem.

Thank you.

**Comment by ricardobaeta** [comment]
Hi @znegrin Thanks! 

Regarding the Google Calendar, I believed it was possible. No harm was done, I'll update the Pattern Library accordingly.

**Comment by marnovo** [comment]
On the GCalendar thing, some sources I read say the "cleanest" way is implementing through alternatives as https://fullcalendar.io/ that's customizable.

Nonetheless, this is a rather minor thing that's not worth our time nor adding a new dependency ATM.

**Comment by dpordomingo** [comment]
deployed into Prod as [v4.7.0](https://github.com/src-d/landing/releases/tag/v4.7.0)

**Comment by ricardobaeta** [comment]
Thanks @dpordomingo ✨

**Pull request merged**